kubernetes_state/datadog_checks/kubernetes_state/kubernetes_state.py

```diff
@@ -398,6 +410,18 @@ def _trim_job_tag(self, name):
         pattern = r"(-\d{4,10}$)"
         return re.sub(pattern, '', name)
 
+    def _extract_job_timestamp(self, name):
+        """
+        Extract timestamp of job names
+        """
+        ts = name.split('-')[-1]
+        if ts.isdigit():
+            return int(ts)
+        else:
+            msg = 'Cannot extract ts from job name {}'.format(name)
+            self.log.debug(msg)
```

Comment:
It's better to do `log.debug(msg, name)` to not format when the logging level isn't debug.